### PR TITLE
Code Review - Week 2

### PR DESCRIPTION
**Pull request by lester-lee**
Look at files changed for feedback!

### CODE REVIEW

File: frontend/src/components/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+/**
+ * Component for the sidebar
+ * @component shows a sidebar that shows users different links
+ */
+const Sidebar = () => {
+  return (
+    <div>
```
Comment (lester-lee) [inline frontend/src/components/Sidebar.js:10]: `<aside>`

File: frontend/src/components/SymptomForm.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      })
+    );
+    navigate("/profile");
+  };
+
+  useEffect(() => {
+    dispatch(me());
+  }, [dispatch]);
+
+  const symptoms = [
```
Comment (lester-lee) [inline frontend/src/components/SymptomForm.js:38]: Similar to a comment about the food options - this should be pulled from the database

File: frontend/src/store/authSlice.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+ * async thunk that authorizes a user through an AJAX request
+ * receives the token from local storage when user is logged in
+ * @returns authentication confirmation
+ * @catches error if database request goes wrong
+ */
+export const me = createAsyncThunk("auth/me", async (thunkAPI) => {
+  const token = window.localStorage.getItem(TOKEN);
+  try {
+    if (token) {
+      const { data } = await axios.get("http://localhost:5000/auth/me", {
```
Comment (lester-lee) [inline frontend/src/store/authSlice.js:15]: Start refactoring these `axios` requests so they don't point to `localhost` anymore - you can put the url in a `.env` that toggles based on whether it's in dev or production

File: frontend/src/store/entrySlice.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const addEntry = createAsyncThunk("user/addFood", async ({ username, entry }) => {
```
Comment (lester-lee) [inline frontend/src/store/entrySlice.js:4]: Would love to see the nice jsdoc comments from `authSlice` :)

File: frontend/src/store/symptomSlice.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const addSymptomEntry = createAsyncThunk(
+  "post symptom entry",
```
Comment (lester-lee) [inline frontend/src/store/symptomSlice.js:5]: Redux has [suggestions on action names](https://redux.js.org/style-guide/#write-action-types-as-domaineventname)

**Review by lester-lee — COMMENTED** [review]
Comments in code!

File: backend/scripts/foods_fix.py
[the remark this answers is not in the document]
Comment (ltreidler) [reply backend/scripts/foods_fix.py:1]: it was for adjusting our food schema. I can delete it now though

**Pull request closed without merging**